roles/edpm_neutron_ovn/tasks/install.yml

```diff
@@ -0,0 +1,34 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Create persistent directories
+  ansible.builtin.file:
+    path: "{{ item.path }}"
+    state: directory
+    setype: "container_file_t"
+    mode: "{{ item.mode | default(omit) }}"
+  loop:
+    - {'path': "{{ edpm_neutron_ovn_agent_config_dir }}"}
+    - {'path': "/var/log/containers/neutron"}
+
+- name: Enable virt_sandbox_use_netlink for healthcheck
```

Comment:
"healthcheck"? is it a proper description of the goal here? I think the idea is that the agent issues direct `ip link` commands on VF representors (or PF? not sure) and hence needs access to this kernel capability. No?

I see the same description used in this repo and in other tripleo adjacent repos for different services, and I suspect it's a remnant from an initial copy-paste that now spread uncontrollably.